includes/admin/meta-boxes/class-wc-meta-box-coupon-data.php

```diff
@@ -116,7 +116,8 @@ public static function output( $post ) {
 						'value'             => esc_attr( $expiry_date ),
 						'label'             => __( 'Coupon expiry date', 'woocommerce' ),
 						'placeholder'       => 'YYYY-MM-DD',
-						'description'       => '',
+						'description'       => __( 'The coupon will expire at 00:00:00 of this date.', 'woocommerce' ), 
```

Comment:
can you remove the whitespace from end of this line?

Reply:
I just commited a fix for whitespace, thank you